obspy/io/dmx/tests/test_core.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+The obspy_dmx.core test suite.
```

Comment:
nit-picky but this mix of underscores and dots is weird

Reply:
leftover from the plugin, replaced now.